.travis.yml

```diff
@@ -3,40 +3,67 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "4"
+  - "6"
 
 sudo: false
+dist: trusty
 
+addons:
+  chrome: stable
+<% if (yarn) { %>
```

Comment:
I think these conditionals are causing problems with the parsing in CI. I think you should drop them in favor of the `yarn` side of each...